Update the procedures in package management (#2117)
theforeman/foreman-documentation: guides/common/modules/proc_upgrading-packages-on-a-host.adoc

[id="upgrading-packages-on-a-host_{context}"]
= Upgrading Packages on a Host

> The package upgrade process takes place via Katello agent or remote execution, depending on your configuration.

Reviewer: ```suggestion
The packages are upgraded through Katello agent or remote execution, depending on your configuration.
```

.Procedure
. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
. Click the name of the host you want to modify.
. Go to *Content* tab, then click the *Packages* tab.

> . Select the *Upgradable* filter from the *Status* box to list the upgradable packages.
> . Click *Upgrade*.
> 
> If a package has more than one available upgrade version, only the latest upgradable version is displayed.

Reviewer: ```suggestion
. Select the *Upgradable* filter from the *Status* box to list the upgradable packages.
+
If a package has more than one available upgrade version, only the latest upgradable version is displayed.
. Click *Upgrade*.
```

> You can run a remote execution job immediately, or select to customize the remote execution job.

Author: What can we add further? Because remote execution is explained separately at another place. And, it has mainly two ways (SSH, Ansible) in customizable jobs for which end-users are well aware during this process.

> You can run a remote execution job immediately, or select to customize the remote execution job.

Reviewer 2: Perhaps not elaborate on REX, but on the upgrade options.
When the user clicks the dropdown menu of the **Upgrade** button, there are two options:

- **Upgrade via remote execution** -- to run remote execution immeditaly
- **Upgrade via customized remote execution** -- to customize the REX job

If the user just clicks the **Upgrade** button, the first option is what happens.

> You can run a remote execution job immediately, or select to customize the remote execution job.

Author: We can document about `Katello Agent` as well. Or remove the same from the introductory paragraph.

> You can run a remote execution job immediately, or select to customize the remote execution job.

Author: > Perhaps elaborate not on REX, but on the upgrade options. When the user clicks the dropdown menu of the **Upgrade** button, there are two options:
> 
>     * **Upgrade via remote execution** -- to run the REX job immediately
> 
>     * **Upgrade via customized remote execution** -- to customize the REX job
> 
> 
> If the user just clicks the **Upgrade** button, the first option is what happens.

Covered this :+1:

[id="upgrading-packages-on-a-host_{context}"]
= Upgrading Packages on a Host

Use this procedure to upgrade packages on a host using the {ProjectWebUI}.
The packages are upgraded through Katello agent or remote execution, depending on your configuration.

.Procedure
. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
. Click the name of the host you want to modify.

> . Go to *Content* tab, then click the *Packages* tab.

Reviewer 2: ```suggestion
. Select the *Content* tab, then select the *Packages* tab.
```

Reviewer: Actually, I would prefer going with what was discussed in https://github.com/theforeman/foreman-documentation/pull/2117#discussion_r1156048203
```suggestion
. On the *Content* tab, navigate to the *Packages* subtab.
```
